autoPyTorch/pipeline/components/setup/network_backbone/ResNetBackbone.py

```diff
@@ -289,13 +290,21 @@ def _build_block(self, in_features: int, out_features: int) -> nn.Module:
             if self.config['use_batch_norm']:
                 layers.append(nn.BatchNorm1d(in_features))
             layers.append(self.activation())
+        else:
+            # if start norm is not None and skip connection is None
+            # we will never apply the start_norm for the first block,
+            # which is why we should account for this case.
+            if not self.config['use_skip_connection']:
+                layers.append(nn.BatchNorm1d(in_features))
```

Comment:
should this not have a condition to check if we should use batch norm?

Reply:
Ups, I missed it. Very good catch.